tests/code_coverage/coverage.test.js

```diff
@@ -296,10 +321,10 @@ function executeFuzzTest(
 		options.push("custom-hooks");
 	}
 	if (coverage) {
-		options.push("--coverage");
+		options.push("--cov");
```

Comment:
Now Jest users will have to change between "--coverage" and "--cov" to show coverage for regular tests and fuzz tests. And the reporters. While the other coverage-related Jest arguments are left unchanged (I think there are several more). 
I wonder how this was solved for JUnit?

Reply:
A suggestion by Norbert was to have the "--cov"-flags alongside "--coverage" flags.